src/datagrid/DataGrid.stories.tsx

```diff
@@ -29,9 +30,39 @@ import {
     pageFilterFunction,
     hideableColumns,
     hideShowColFooter,
+    selectedRows,
+    getRowData,
 } from "./DataGridValues";
 import {CustomFilter} from "./CustomFilter";
+import {DataGridRow} from "./DataGrid";
 
+const store = new Store({
+    selectedRows: selectedRows,
+    rows: paginationRows.slice(0, 5),
+    selectRowCallback: (row?: DataGridRow) => {
+        const rowID = row && row.rowData[0].cellData;
+        const index = selectedRows.indexOf(rowID);
+        if (row && row.isSelected) {
```

Comment:
In storybook,on selecting all followed by deselection of one, the selected row count generated is not matching with the total selected rows.

Reply:
@payald Since it is just a storybook I have not taken care of all possible scenarios for updating count. We need this logic in Parent component who will use datagrid and pass this count.